VAMobile/e2e/tests/PersonalInformationScreen.e2e.ts

```diff
@@ -98,11 +98,13 @@ describe('Personal Information Screen', () => {
 
     await expect(element(by.text(PersonalInfoConstants.PERSONAL_INFORMATION_TEXT)).atIndex(0)).toExist()
     await expect(element(by.text('Preferred name saved'))).toExist()
-    await expect(element(by.text('Kimberlee'))).toExist()
+    //Commented out until user endpoint is completed
+    //await expect(element(by.text('Kimberlee'))).toExist()
```

Comment:
This can be uncommented now that the demographics demo mode PR is merged. Same for the other parts of this file